config/jobs/kubernetes/sig-scalability/sig-scalability-golang.yaml

```diff
@@ -40,7 +40,6 @@ periodics:
       - runner.sh
       args:
       - make
-      - --
```

Comment:
I don't think this was necessary before either, but it was dropped by `execute.py`.

None of the other jobs in the previous PR appear to have a bare `--` argument seperator.